browser/lib/transport/withoutjsonp.js

```diff
@@ -0,0 +1,7 @@
+import XHRPollingTransport from './xhrpollingtransport';
```

Comment:
Could you consider adding some commentary to the top of this file to explain why it's here and what it does for the wider project as a whole? It's a bit opaque right now.

Reply:
Good point, I've added a header in abeda9a.

Reply:
Thanks. Perfect.